third_party/build/bin/ryu_build.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# package ryu with magma patches
+
+set -e
```

Comment:
- Might be good to add `set -euo pipefail`.
- Is this script just for documentation or will this be the standard way to create the package?

Reply:
have you tried if this works in the current setup, i.e., `/third_party/build$ ./build.py ryu`?

Reply:
No, I don't have a prod VM (on the magma_vm the `ryu_build.sh` does not exist).

Reply:
discussed offline - does not work with util.sh. Refactoring is out of scope here.